frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/utils.ts

```diff
@@ -1,33 +1,35 @@
-import * as _ from 'lodash';
 import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
-import { PrometheusHealthHandler } from '@console/plugin-sdk';
+import { PrometheusHealthHandler, ResourceHealthHandler } from '@console/plugin-sdk';
 import { getResiliencyProgress } from '../../../../utils';
+import { ResourcesObject } from '@console/internal/components/utils/k8s-watch-hook';
+import { K8sResourceKind } from '@console/internal/module/k8s';
 
-const CephHealthStatus = [
-  {
+const CephHealthStatus = {
+  HEALTH_OK: {
     state: HealthState.OK,
   },
-  {
+  HEALTH_WARN: {
     state: HealthState.WARNING,
   },
-  {
+  HEALTH_ERROR: {
     state: HealthState.ERROR,
   },
-  {
-    state: HealthState.NOT_AVAILABLE,
-  },
-];
+};
 
-export const getCephHealthState: PrometheusHealthHandler = (responses = [], errors = []) => {
-  if (errors[0]) {
-    return CephHealthStatus[3];
+export const getCephHealthState: ResourceHealthHandler<ResourcesObject> = (resourceObject) => {
```

Comment:
we can be more specific with the type
```suggestion
export const getCephHealthState: ResourceHealthHandler<{ceph: K8sResourceKind[]}> = ({ceph: {data, loaded, loadError}}) => {
```